diff --git a/node_modules/react-native-aes-gcm-crypto/android/build.gradle b/node_modules/react-native-aes-gcm-crypto/android/build.gradle
index 360dffb..0a2e96c 100644
--- a/node_modules/react-native-aes-gcm-crypto/android/build.gradle
+++ b/node_modules/react-native-aes-gcm-crypto/android/build.gradle
@@ -66,7 +66,6 @@ android {
 
 repositories {
   mavenCentral()
-  jcenter()
   google()
 
   def found = false
